integration_tests/src/main/python/parquet_test.py

```diff
@@ -193,11 +201,28 @@ def test_ts_read_fails_datetime_legacy(gen, spark_tmp_path, ts_write, ts_rebase,
             lambda spark : readParquetCatchException(spark, data_path),
             conf=all_confs)
 
+
+# https://github.com/NVIDIA/spark-rapids/issues/1324
```

Comment:
nit: I would prefer to have us describe what we are testing instead of just adding an issue number.  The issue number is fine for xfail because it provides tracking so I can see if the issue is fixed and we can go back to look if the test is now passing.  Here I'm not even sure we need a comment because the name of the test explains a lot of this.

Reply:
Deleted.